Client/mods/deathmatch/logic/CClientCamera.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <StdInc.h>
 #include <game/CCam.h>
+#include "../../game_sa/CCameraSA.h"
```

Comment:
I don’t like this. In general, we try not to mix projects with each other. The overall layer is like this:

Client Deathmatch -> Client SDK (virtual) -> CGameSA

So in `CCamera.h` you can add a virtual method that:

Returns an interface or a bool indicating whether it exists.

The same goes for FOV - instead of `pCameraInterface->FOVDuringInter;` you can create a method and use it.